apps/subscriptions/src/data/filters.ts

```diff
@@ -0,0 +1,106 @@
+import type { FiltersInstructions } from '@commercelayer/app-elements'
+
+export const instructions: FiltersInstructions = [
+  {
+    label: 'Markets',
+    type: 'options',
+    sdk: {
+      predicate: 'market_id_in'
+    },
+    render: {
+      component: 'inputResourceGroup',
+      props: {
+        fieldForLabel: 'name',
+        fieldForValue: 'id',
+        resource: 'markets',
+        searchBy: 'name_cont',
+        sortBy: { attribute: 'name', direction: 'asc' },
+        previewLimit: 5,
+        filters: {
+          disabled_at_null: true
+        }
+      }
+    }
+  },
+  {
+    label: 'Status',
+    type: 'options',
+    sdk: {
+      predicate: 'status_in'
+    },
+    render: {
+      component: 'inputToggleButton',
+      props: {
+        mode: 'multi',
+        options: [
+          { value: 'draft', label: 'Draft' },
+          { value: 'active', label: 'Active' },
+          { value: 'inactive', label: 'Inactive' },
+          { value: 'cancelled', label: 'Cancelled' }
+        ]
+      }
+    }
+  },
+  {
+    label: 'Frequency',
+    type: 'options',
+    sdk: {
+      predicate: 'frequency_in'
+    },
+    render: {
+      component: 'inputToggleButton',
+      props: {
+        mode: 'multi',
+        options: [
+          {
+            value: 'hourly',
+            label: 'Hourly'
+          },
+          {
+            value: 'daily',
+            label: 'Daily'
+          },
+          {
+            value: 'weekly',
+            label: 'Weekly'
+          },
+          {
+            value: 'monthly',
+            label: 'Monthly'
+          },
+          {
+            value: 'two-month',
+            label: '2-Month'
```

Comment:
I would improve those label, what do you think of using the same here? https://github.com/commercelayer/mfe-checkout/blob/bb3714ef8f8e97c641b9015736b8f28d9f27139c/public/static/locales/en/common.json#L59

Reply:
Sure!